rasa_nlu/classifiers/sklearn_intent_classifier.py

```diff
@@ -58,6 +58,10 @@ class SklearnIntentClassifier(Component):
         # C parameter of the svm - cross validation will select the best value
         "C": [1, 2, 5, 10, 20, 100],
 
+        # gamma parameter of the svm - cross validation will
+        # select the best value
+        "gamma": [0.00001, 0.0001, 0.001, 0.1, 1, 10, 100, 1000],
```

Comment:
I am a bit concerned that doing a grid search with 8 values will 8x the current training time. I think for the moment it would be better to set it to one value, still allowing the user to specify more but keeping the current training time.